src/EnergyPlus/ThermalISO15099Calc.cc

```diff
@@ -116,43 +116,47 @@ void film(Real64 const tex, Real64 const tw, Real64 const ws, int const iwd, Rea
     Real64 bexp;
 
     // calculation of convection component of exterior film coefficient using the :
-    {
-        auto const SELECT_CASE_var(ibc);
-        if (SELECT_CASE_var == 0) { // ISO 15099
-            hcout = 4.0 + 4.0 * ws;
-        } else if (SELECT_CASE_var == -1) { // old ASHRAE SPC142 correlation
-            if (iwd == 0) {                 // windward
-                if (ws > 2.0) {
-                    vc = 0.25 * ws;
-                } else {
-                    vc = 0.5;
-                }
-            } else { // leeward
-                vc = 0.3 + 0.05 * ws;
-            }
-            hcout = 3.28 * std::pow(vc, 0.605);
-            hcout *= conv;                  // convert to metric
-        } else if (SELECT_CASE_var == -2) { // Yazdanian-Klems correlation:
-            if (iwd == 0) {                 // windward
-                acoef = 2.38;
-                bexp = 0.89;
-            } else { // leeward
-                acoef = 2.86;
-                bexp = 0.617;
+    switch (ibc) {
+    case 0: { // ISO 15099
```

Comment:
Do these things have symbolic names somewhere? Or are we just using a combination of integers and comments?